Move lineage AWS IAM permissions into compute env and manual setup pages (#1401)
seqeralabs/docs: platform-cloud/docs/compute-envs/aws-cloud.md

}
```

### Data lineage (optional)

> If [data lineage](../data/data-lineage) is enabled in your workspace, the Platform integration credentials require the following additional permissions to create the queue infrastructure and bucket notifications used by the lineage service:

Reviewer: ```suggestion
If you enable [data lineage](../data/data-lineage) in your workspace, add the following permissions to your Platform integration credentials to create the queue infrastructure and bucket notifications used by the lineage service:
```

```json
{
  "Sid": "LineageIntegrationSQS",
  "Effect": "Allow",
  "Action": [
    "sqs:CreateQueue",
    "sqs:GetQueueAttributes",
    "sqs:SetQueueAttributes",
    "sqs:GetQueueUrl",
    "sqs:ReceiveMessage",
    "sqs:DeleteMessage"
  ],
  "Resource": "arn:aws:sqs:<REGION>:<ACCOUNT_ID>:seqera-lineage-*"
},
{
  "Sid": "LineageIntegrationS3",
  "Effect": "Allow",
  "Action": [
    "s3:CreateBucket",
    "s3:GetBucketNotificationConfiguration",
    "s3:PutBucketNotificationConfiguration",
    "s3:GetBucketLocation"
  ],
  "Resource": "arn:aws:s3:::seqera-lineage-*"
}
```

> If you manage your own EC2 instance role (rather than letting Seqera create it automatically), attach the additional S3 policy described in [Manual AWS Batch configuration](../enterprise/advanced-topics/manual-aws-batch-setup#create-an-ec2-instance-role) to that role.

Reviewer: ```suggestion
If you manage your own EC2 instance role (rather than letting Seqera create it automatically), see [Manual AWS Batch configuration](../enterprise/advanced-topics/manual-aws-batch-setup#create-an-ec2-instance-role) for the additional S3 policy to attach to that role.
```
